panther_analysis_tool/main.py

```diff
@@ -1352,6 +1386,13 @@ def setup_parser() -> argparse.ArgumentParser:
         "help": "A destination name that may be returned by the destinations function. "
         "Repeat the argument to define more than one name.",
     }
+    sort_test_results_name = "--sort-test-results"
+    sort_test_results_arg: Dict[str, Any] = {
+        "required": False,
+        "default": False,
+        "type": strtobool,
+        "help": "Print detection test results first by passed/errored outcome, and then by rule ID"
+    }
```

Comment:
Could you make it so that to opt into this behavior you just need to pass `--sort-test-results` rather than `--sort-test-results true`?

We do this for `--skip-disabled-tests` and `--skip-tests`, for example: https://github.com/panther-labs/panther_analysis_tool/blob/5b8c955a922242f64b432600abe11fe448ed9523/panther_analysis_tool/main.py#L1313-L1326

Reply:
@cdzombak 

I've made updates to avoid printing the blank line when results are being sorted.
I've also updated the `--sort-test-results` arg implementation as you asked. 
Note: This seems to be done inconsistently for some flags currently. For example, I was following the implementation of the `--ignore-extra-keys` arg.

Reply:
Thanks, @slw07g!

> Note: This seems to be done inconsistently for some flags currently. For example, I was following the implementation of the `--ignore-extra-keys` arg.

That's definitely true. I bet you can see that PAT has grown organically/accidentally into the tool it is today, for better or worse 😆 

We'd like to standardize these, but it's not a high priority in comparison to other stuff our team is working on. (I wonder, too, if it's possible to have those flags accept an argument _optionally_, to avoid it being a breaking change — I haven't looked into that yet.)